### PR TITLE
fix(dashboard): fix circle ci

### PR DESCRIPTION
**Pull request by d4x1**
### ⚠️ Pre Checklist

> Please complete _ALL_ items in this checklist, and remove before submitting

- [x] I have read through the [Contributing Documentation](https://devlake.apache.org/community/).
- [x] I have added relevant tests.
- [x] I have added relevant documentation.
- [x] I will add labels to the PR, such as `pr-type/bug-fix`, `pr-type/feature-development`, etc.

### Summary
What does this PR do?
1. Fix wording errors in CircleCI's dashboard
2. Update filter 

### Does this close any open issues?
Closes N/A

### Screenshots
Include any relevant screenshots here.
#### Filter
image

#### Wording
image
image

### Other Information
Any other information that is important to this PR.

### CODE REVIEW

File: grafana/dashboards/CircleCI.json [hunk 1]
```diff
@@ -947,18 +947,18 @@
...
         },
         "datasource": "mysql",
-        "definition": "select distinct cicd_scope_id as text from cicd_pipelines where id like \"%circleci%\" ",
+        "definition": "select concat(name, '--', id) as text from cicd_scopes where id like \"%circleci%\" ",
         "description": null,
         "error": null,
         "hide": 0,
         "includeAll": true,
-        "label": "Job Name",
+        "label": "Project Name",
```
Comment (Startrekzky) [inline grafana/dashboards/CircleCI.json:955]: Maybe 'CircleCI Project'?

File: grafana/dashboards/CircleCI.json [hunk 2]
```diff
@@ -358,12 +358,12 @@
...
       "timeFrom": null,
       "timeShift": null,
-      "title": "3. Total Build Result Distribution",
+      "title": "3. Total Workflow Result Distribution",
       "type": "piechart"
     },
     {
       "datasource": "mysql",
-      "description": "Number of successful builds / Number of total builds",
+      "description": "Number of successful pipelines / Number of total pipelines",
```
Comment (Startrekzky) [inline grafana/dashboards/CircleCI.json:366]: Not updated yet

File: grafana/dashboards/CircleCI.json (hunk 1 above)
Comment (d4x1) [reply grafana/dashboards/CircleCI.json]: Updated.

File: grafana/dashboards/CircleCI.json (hunk 2 above)
Comment (d4x1) [reply grafana/dashboards/CircleCI.json]: Updated.

File: grafana/dashboards/CircleCI.json [hunk 3]
```diff
@@ -532,19 +532,19 @@
...
         }
       ],
-      "title": "1.1 Total Number of Successful Builds [Each Month]",
+      "title": "1.1 Total Number of Successful Workflows [Each Month]",
       "type": "barchart"
     },
     {
       "datasource": "mysql",
-      "description": "1. Mean Build success rate over time.\n2. The builds being calculated are filtered by \"build starting time\" (time filter at the upper-right corner) and \"Jira board\" (\"Choose Board\" filter at the upper-left corner)",
+      "description": "1. Mean workflow runs success rate over time.\n2. The workflow runs being calculated are filtered by \"build starting time\" (time filter at the upper-right corner) and \"Jira board\" (\"Choose Board\" filter at the upper-left corner)",
```
[the remark this answers is not in the document]
Comment (d4x1) [reply grafana/dashboards/CircleCI.json]: Done.

**Review by Startrekzky — APPROVED** [review]
LGTM

**Pull request merged**